tests/playwright/tests/forms/form-generator.spec.ts

```diff
@@ -0,0 +1,163 @@
+import { test, expect } from "@playwright/test";
+import { fillExistingFields, acceptCookiePolicy, extractAllFormUrls } from "../../helpers/commands.ts";
+import { formTextFields, formCheckboxFields } from "../../helpers/form-fields.ts";
+
+const openModal = async (page) => {
+  await page.goto("/tests/_form-generator");
+  await acceptCookiePolicy(page);
+  const contactUsLink = page.locator('a[aria-controls="contact-modal"]');
+  await expect(contactUsLink).toBeVisible();
+  await contactUsLink.click();
+  const modal = page.locator("#contact-modal");
+  await expect(modal).toBeVisible();
+}
+
```

Comment:
```suggestion

test.beforeEach(async ({ page }) => {
  await openModal(page); 
})

```
Since all the tests are first opening the modal, we don't need to repeat that step manually in every test. Instead, we can do something like this:
(Make sure to remove `await openModal(page);` call from every test.